src/atomist.config.ts

```diff
@@ -26,15 +21,10 @@ export const configuration: Configuration = {
         HelloWorld,
         HelloChannel,
         HelloHttp,
-        SpringBootVersionReviewer,
-        VersionSpreadReviewer,
-        VersionMapper,
         NewAutomation,
-        SpringBootModernizer,
-        UpdateCopyright,
-        ReviewCopyright,
+        () => updateCopyrightInOneRepository(),
         // Use a factory if you like...
-        () => new SpringBootVersionUpgrade(),
+        () => new ReviewCopyright(),
```

Comment:
Maybe this should be changed to use the auto-scanning?

Reply:
maybe, but as a separate PR... as a sample I kinda like it being explicit though, it helps _people_ find all the automations